src/pe_reports/home/views.py

```diff
@@ -2,10 +2,14 @@
 
 Automate the process to add stakeholder information to Cyber Sixgill portal.
 """
+# Standard Python Libraries
+import logging
 
 # Third-Party Libraries
 from flask import Blueprint, render_template
 
+LOGGER = logging.getLogger(__name__)
```

Comment:
In some files, you call this `LOGGER` and in others you call it `logger`.  You should pick one and use it consistently throughout this repository.

Reply:
@dav3r , 
I have updated the call to LOGGER to be consistent throughout the repo. The changes were made per commit [d11f89c](https://github.com/cisagov/pe-reports/pull/169/commits/d11f89c78939b26ed8344d0ad355b62febfc21d8)